docs/ocis/adr/0009-routes.md

```diff
@@ -0,0 +1,81 @@
+## Problem Statement
```

Comment:
needs the adr header ;-)

Reply:
Yes; the current state is a braindump, I'll format it to the right format when I'm past beyond this stage :)